pkg/condition/condition.go

```diff
@@ -82,6 +83,7 @@ func (c *EvaluableCondition) compile() error {
 
 	prgopts := []cel.ProgramOption{
 		cel.EvalOptions(cel.OptPartialEval),
+		cel.EvalOptions(cel.OptExhaustiveEval),
```

Comment:
> // OptExhaustiveEval causes the runtime to disable short-circuits and track state.

why do we need this?

Reply:
We don't. I removed it. It was residual.